src/settings.ts

```diff
@@ -131,6 +132,11 @@ export function settingsFromWorkspace(): ISettings {
     enable: true,
   };
 
+  const defaultDockerSettings: IEditorServiceDockerSettings = {
+    imageName: "linguapupuli/puppet-langauge-server",
```

Comment:
This includes the misspelled image, will probably want to fix that before we ship.